### PR TITLE
Use TimeSpan for break period

### LINKED ISSUE #7826: Storage: Use TimeSpan for LeaseClient.Break's breakPeriod
We should match what we did for the lease duration.

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Fixes #7826

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs [hunk 1]
```diff
@@ -1018,7 +1019,7 @@ private void EnsureClient()
                         return (await BlobRestClient.Blob.BreakLeaseAsync(
                             Pipeline,
                             Uri,
-                            breakPeriod: breakPeriodInSeconds,
+                            breakPeriod: serviceBreakPeriod,
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs]: Does this need a `.TotalSeconds` or similar?

File: sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs [hunk 2]
```diff
@@ -1040,7 +1041,7 @@ private void EnsureClient()
                         return (await BlobRestClient.Container.BreakLeaseAsync(
                             Pipeline,
                             Uri,
-                            breakPeriod: breakPeriodInSeconds,
+                            breakPeriod: serviceBreakPeriod,
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs]: Same here?

File: sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs]: `serviceBreakPeriod` is a local var where I've already converted to seconds.

Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs]: I'm not sure how I missed that...  but yes, yes it is.

**Review by tg-msft — APPROVED** [review]

**Comment by JoshLove-msft** [comment]
@amnguye @seanmcc-msft I would like to merge this in by EOD Wednesday, if possible. Please if you can take a look, thanks!

**Review by amnguye — APPROVED** [review]

**Review by amnguye — APPROVED** [review]

**Pull request merged**